include/arch/syspage-imxrt.h

```diff
@@ -20,13 +20,13 @@
 
 typedef struct {
 	struct {
-		u32 type;
-		u32 allocCnt;
+		unsigned int type;
+		unsigned int allocCnt;
 		struct {
-			u32 rbar;
-			u32 rasr;
+			unsigned int rbar;
+			unsigned int rasr;
 		} table[16] __attribute__((aligned(8)));
-		u32 map[16]; /* ((u32)-1) = map is not assigned */
+		unsigned int map[16]; /* ((unsigned int)-1) = map is not assigned */
```

Comment:
Unrelated change and I'm not sure if we want to change it that way

Reply:
You have right that it is not related to the context of this PR but I have added appropriate comment in commit message. 
`u32` and `u8` are described in `hal`. This code is shared between plo and kernel so it will be better to not use types which are defined in both modules separately.

Reply:
Ok for now, but imho this types might be/should be shared between plo and kernel in the future